components/inputs/test/input-search.html

```diff
@@ -37,13 +37,9 @@
 				let elem, input;
 
 				async function getFixture(name) {
-					return new Promise((resolve) => {
```

Comment:
Aside - the old code didn't look quite right - the `async` was originally not needed.